### DISCUSSION TITLE
Set Stars Lists private

### DISCUSSION THREAD

**Discussion by davidleitw** [community/community#16473] [Other Feature Feedback, Questions, & Ideas]
When I star some repositories into my lists, other one can see my List name, I think that the stars lists can be private, thanks.

**Comment by britishleo** [comment]
> When I star some repositories into my lists, other one can see my List name, I think that the stars lists can be private, thanks.

I Agree With You

**Reply by chichi88lyang** [reply]
Very important observation

**Comment by GoaGill** [comment]
davidleitw ***@***.***> schrieb am Do., 12. Mai 2022, 20:24:
